portality/scripts/link_checker_report.py

```diff
@@ -67,7 +133,17 @@ def fetch_matching_rows(df, report_values):
         return pd.DataFrame()
 
 
-def check_links(df, journal_df):
+def _index_journals(df):
+    jidx = {}
+    for index, row in df.iterrows():
+        for cell in row:
+            # FIXME: assumes each URL only appears once
+            if isinstance(cell, str) and cell.startswith("http"):
+                jidx[cell] = (row[0], row[50], row[51], row[54])
```

Comment:
Please annotate the structure of the `jidx`

Reply:
Documentation added in commit